app/views/donors/show.html.erb

```diff
@@ -1,7 +1,26 @@
 <div class="content-wrapper-narrow content-container">
-  <h3>
+  <section class="donor-content-section">
+    <h3>
+      <span class="supplier-name"><%= @donor.name %> (<%=@donor.email %>)</span>
+    </h3>
+
+    <h2 class="section-label">
+      <%= t(".address") %>
+    </h2>
+    <span><%= @donor.address %></span>
+
+    <h2 class="section-label">
+      <%= t(".grow_information") %>
+    </h2>
+    <span><%= t(".grow_information_sentence",
+            is_is_not: is_or_isnt_on_site(@donor),
```

Comment:
What do you think about another I18n key that is more descriptive and less redundant?

What do you think about `on_site`?